app/config/messaging.js

```diff
@@ -1,4 +1,5 @@
 const sharedConfig = {
+  protectiveMonitoringUrl: process.env.PROTECTIVE_MONITORING_URL,
```

Comment:
I think this would make more sense in the general config file.

This `sharedConfig` is for config shared between the message queues/topics

Reply:
Make sense. Thanks